File: LaundryLoop.Api/Models/CartItem.cs
using System.ComponentModel.DataAnnotations;

namespace LaundryLoop.Api.Models;

public class CartItem
{
    public int Id { get; set; }

    [Required]
    public int BookingId { get; set; }

    [Required, MaxLength(50)]
    public string ItemType { get; set; } = string.Empty; // 'laundry', 'addon', 'delivery'

    [Required, MaxLength(100)]
    public string ItemName { get; set; } = string.Empty;

    [MaxLength(200)]
    public string? Description { get; set; }

    [Required]
    public decimal Price { get; set; }

    // Navigation
    public Booking? Booking { get; set; }
}

File: LaundryPickupDeliveryPHP/src/Models/CartItem.php
<?php

declare(strict_types=1);

namespace LaundryLoop\Models;

class CartItem
{
    public int     $id          = 0;
    public int     $bookingId   = 0;
    public string  $itemType    = '';   // 'laundry', 'addon', 'delivery'
    public string  $itemName    = '';
    public ?string $description = null;
    public float   $price       = 0.0;

    public static function fromRow(array $row): self
    {
        $c              = new self();
        $c->id          = (int)   ($row['Id']          ?? 0);
        $c->bookingId   = (int)   ($row['BookingId']   ?? 0);
        $c->itemType    = (string) ($row['ItemType']    ?? '');
        $c->itemName    = (string) ($row['ItemName']    ?? '');
        $c->description =           $row['Description'] ?? null;
        $c->price       = (float)  ($row['Price']       ?? 0.0);
        return $c;
    }

    public function toArray(): array
    {
        return [
            'id'          => $this->id,
            'bookingId'   => $this->bookingId,
            'itemType'    => $this->itemType,
            'itemName'    => $this->itemName,
            'description' => $this->description,
            'price'       => $this->price,
        ];
    }
}
